creme/tree/_split_criterion/info_gain_split_criterion.py

```diff
@@ -4,13 +4,13 @@
 
 
 class InfoGainSplitCriterion(SplitCriterion):
-    """ Information Gain split criterion.
+    """Information Gain split criterion.
 
     A measure of how often a randomly chosen element from the set would be
     incorrectly labeled if it was randomly labeled according to the
     distribution of labels in the subset.
 
-    `Wikipedia entry <https://en.wikipedia.org/wiki/Decision_tree_learning#Gini_impurity>`_
+    `Wikipedia entry <https://en.wikipedia.org/wiki/Decision_tree_learning#Information_gain>`_
```

Comment:
Change link format to markdown

Reply:
Changed in commit [2a55a14](https://github.com/creme-ml/creme/pull/362/commits/2a55a14fa6f2b98d3160bd46f55f85d9fb6f272c).